### PR TITLE
Add missing AWS credentials

### PR DESCRIPTION
**Pull request by sbrunner**
Related to: https://jira.camptocamp.com/browse/GEO-3071

### CODE REVIEW

File: doc/integrator/tilegeneration.rst [hunk 1]
```diff
@@ -122,3 +122,9 @@ If you setup all the default options, you can generate the tiles by using the co
 .. prompt:: bash
 
     docker-compose exec tilecloudchain generate_tiles
+
+AWS credentials
+~~~~~~~~~~~~~~~
+
+To be able to connect to the S3 service sou should define the folowing variable in the ``.env.mako`` file:
```
Comment (jwkaltz) [inline doc/integrator/tilegeneration.rst]: service sou -> service, you
folowing variable -> following variables

Comment (sbrunner) [reply doc/integrator/tilegeneration.rst]: Thanks :-)

File: doc/integrator/tilegeneration.rst [hunk 2]
```diff
@@ -122,3 +122,21 @@ If you setup all the default options, you can generate the tiles by using the co
...
+
+To be able to connect to the S3 service, you should define the following variables in the ``.env.mako``
+file:
+
+code::
+
+  AWS_ACCESS_KEY_ID=<access_key_id>
+  AWS_SECRET_ACCESS_KEY=<secret_access_key>
+
+If you don't want to commit thoes credentials you can add them in your ``~/.bashrc`` file:
```
Comment (jwkaltz) [inline doc/integrator/tilegeneration.rst]: don't -> do not
thoes -> these

Comment (sbrunner) [reply doc/integrator/tilegeneration.rst]: Thanks :-)

**Pull request merged**